.tekton/hypershift-operator-main-pull-request.yaml

```diff
@@ -206,34 +206,6 @@ spec:
             workspace: workspace-arm64
           - name: basic-auth
             workspace: git-auth
-      - name: clone-repository-s390x
```

Comment:
Can we just comment everything out rather than removing it? That way we can just uncomment it back in once s390x is supported again.

Reply:
+1

Reply:
Personally I'm not a big fan of commenting out code. Git history is always there and one can see whatever was removed. But sure, if that's what folks want.

Reply:
As agreed, a JIRA issue is now referenced